src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/AggregationSingle.java

```diff
@@ -0,0 +1,51 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation.AggregationType;
+import com.alipay.oceanbase.rpc.util.Serialization;
+
+import static com.alipay.oceanbase.rpc.util.Serialization.encodeObUniVersionHeader;
+import static com.alipay.oceanbase.rpc.util.Serialization.getObUniVersionHeaderLength;
+
+public class AggregationSingle {
```

Comment:
Is **AggregationSingle** a sub-class of **AbstractPayload**? Since you implement `getPayloadSize` below.

Reply:
fixed